yarn.lock

```diff
@@ -3817,7 +3817,6 @@
   version "0.0.0"
   uid ""
 
-
```

Comment:
WTF?!? Why was this even here in the first place??

Reply:
Likely a bad (well, not ideal) conflict resolution attempt

Reply:
:grimacing: https://github.com/elastic/kibana/pull/148128/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR3820